CMakeLists.txt

```diff
@@ -71,25 +71,45 @@ function(mbed_configure_app_target target)
     mbed_set_language_standard(${target})
 endfunction()
 
+# GCC ARM requires preprecessing linker script, execute generators to get definitions needed for
+# this step - linker options and compile definitions 
+if(MBED_TOOLCHAIN STREQUAL "GCC_ARM")
```

Comment:
I'll move these to Gcc Arm toolchain file, this main file will only execute a function .

Reply:
Waiting for that before reviewing then.

Reply:
Done, updated. Much better now :)